src/pom.xml

```
@@ -58,6 +58,12 @@
             <artifactId>spring-boot-actuator</artifactId>
             <version>${spring.version}</version>
         </dependency>
+        <!-- Spring Mongo DB -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-mongodb</artifactId>
+            <version>2.1.3.RELEASE</version>
```

Since it's the starter dependency, it shares the same version as the spring boot, so use the property instead of using the hardcoded version

```suggestion
            <version>${spring.version}</version>
```